decision-making-processes.md

```diff
@@ -1,35 +1,157 @@
+Once there is a Solid Test Suite it could be made a requirement that identity providers, Pod providers, and Solid app providers need to pass the Solid Test Suite to be able to participate in the Solid Panel. 
 
-Each individual in the Decision Panel has one vote even if they have multiple criteria for being in the Decision panel, so each person in the Decision Panel has an equal voice. Each individual can vote on a first and second choice, or they can abstain. After counting the first choice votes, the votes from the bottom three options get transferred to the second choice vote to avoid strategic voting that does not reflect the true opinion of the Decision Panel. The votes are recounted and the majority vote wins. The Solid Manager's responsibilities are to -- 
-* make sure everyone on the Decision Panel is aware of the vote
-* explain how they can vote
-* collect the final results
-* communicate the outcome
+Companies with multiple employees have a vote per individual employee rather than per company. 
 
-# Changes to the Solid Specification 
-Decisions around the Solid Specification changes need to have a procedure around them to encourage conversation to find solutions to more complex decisions. 
+## Appoinment
+Anyone can apply to the Solid Panel as long as they fulfill the criteria above. The Solid Leader appoints candidates to roles. 
+
+# Election Process for Decisions other than Appointing Individuals to Roles 
+* Changes to the Solid specification
+* Changes to the [Solid roadmap](https://github.com/solid/information/blob/master/solid-roadmap.md)
+* Changes to the Solid Test Suite
+* Starting or changing the aim or project manager of a [Solid Project](https://github.com/orgs/solid/projects) i.e. a GitHub project in the Solid Github account
 
 ## Step 1. Making a Suggestion 
-Anyone can suggest a change to the Solid specification by getting a GitHub account and submitting a pull request or issues to the Solid specification repositories. 
+Anyone can suggest a change by getting a GitHub account and submitting a pull request or issues to the relevant repository on the Solid GitHub account. 
 
 ## Step 2. Inviting a Conversation around your Suggestion to Find Consensus
 You will then need to get a W3C account and join the W3C Solid Community Group to raise a conversation about your suggestion as an item on the weekly call agenda. By adding the item to the agenda all members of the W3C Solid Community Group will be automatically notified and therefore are able to comment on the pull request or issue directly. 
 
+Anyone can suggest routes forward to the suggested changes with pros and cons of each route forward on the original pull request or issues. Below is a table format where all the final suggested routes forward, pros and cons should be included. 
+
+| Route Forward | Pros to Consider | Cons to Consider |
+| ------------- | ------------- | ------------- |
+| (insert suggestion)  | (insert suggestion)  | (insert suggestion)  |
```

Comment:
I'm not sure this structure works for everything. I would just leave it as a suggestion.

Reply:
sounds good, have edited text so that the table is presented as a suggested format